src/styles/blocks/stars--2.css

```diff
@@ -0,0 +1,44 @@
+.stars--2 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--2 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of styling `.stars--2 .stars__star:nth-child(1)`, give each star element a class and style that class. This will make your CSS more maintainable and your HTML more readable.